### ISSUE TITLE
Descriptions.md not loading on IIS [Fixed - Sharing info]

### ISSUE THREAD

**Issue by welcomebrand** [NoDivide/astrum#149]
I don't suppose you have many Windows users but reporting this just in case you want to update your install docs for those who may run into this issue. Details and fix below. 

Cheers.

#### Tell us about the issue you are experiencing
When you install Astrum for the first time on a Windows server it's possible that when you view the styleguide you'll get an error saying the library description files won't load. 

#### What behaviour would you expect?
The library should just load as normal.

#### How can this issue be replicated?
Install Astrum on a Windows machine where your site is running on IIS. 

#### Screenshots / Screencast
![patternlibraryerror](https://user-images.githubusercontent.com/337631/33317310-a10a415e-d42e-11e7-8684-68c00e2b8fea.png)

#### Your environment

Platform: Windows 10
OS Version:
Node Version:
NPM Version:
Astrum Version: 1.9.5

## To fix
The cause of this is that Markdown is not a supported MIMETYPE on IIS by default so you need to add this setting into your IIS config. 

[How to edit your MIMETYPE settings on IIS](https://technet.microsoft.com/en-us/library/cc725608(v=ws.10).aspx)
[The Markdown MIMETYPE setting required](https://stackoverflow.com/questions/10701983/what-is-the-mime-type-for-markdown)

Once you've updated this in IIS, you should be able to just refresh your pattern library and it will work as expected.

**Comment by hankchizljaw** [comment]
Thanks for this @welcomebrand, We really appreciate it! 

I'll assign myself to update the installation documentation 🙂

**Comment by hankchizljaw** [comment]
PR created: #150 

Thanks again, @welcomebrand!

**Comment by dsagner** [comment]
I've successfully edited my MIMETYPE setting on IIS and I added the IISMarkdownHandler locally to my .NET MVC project, the markdown content is now getting read and inserted to it's respected placeholders, but before each of them I get a "<!doctype html>" tag rendered whether or not the description.md file contains some text inside it. I have this only locally on my machine, therefore a screenshot is being added to this comment.  I haven't made any changes to the intro.md file for the Introduction page to test if it is present on the generated files after astrum initialization in the project folder. Did I just miss something from the links above or can the cause be something else?

[Screenshot 1](http://prntscr.com/i5vnst)

[Screenshot 2](http://prntscr.com/i5vo3j)
